lte/gateway/docker/docker-compose.yaml

```diff
@@ -359,3 +359,53 @@ services:
       timeout: "4s"
       retries: 3
     command: /var/opt/magma/bin/envoy_controller
+
+  # for docker compose v1: https://github.com/docker/compose/issues/8351#issuecomment-1085811366
+  # also see https://docs.docker.com/compose/compose-file/#depends_on
+  waitForHealthyPods:
```

Comment:
why do we use a completly different approach here to check for a healthy containers as we do in the [tests](https://github.com/magma/magma/blob/638acb51e3c33029be2ecef5a4e4244aef275c87/lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py#L1207)? I am pretty sure you should not directly reuse the code as the s1ap tester lives in a different vm. You could copy the approach.

Reply:
Anyway I think we should merge this as it is a big improvement over the current state of things.